articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+Locate the [Draw](xref:Microsoft.Xna.Framework.Game.Draw(Microsoft.Xna.Framework.GameTime)) method in the *Game1.cs* file and add the following after the [Clear](xref:Microsoft.Xna.Framework.Graphics.GraphicsDevice.Clear(Microsoft.Xna.Framework.Color)) method call is made:
+
+```cs
+_spriteBatch.Begin();
+_spriteBatch.Draw(_logo, Vector2.Zero, Color.White);
+_spriteBatch.End();
+```
+
+These lines of code tell the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) to draw the logo texture you loaded in the top-left corner of the game window. Run the game and you should see something similar to the following:
+
+<figure><img src="./images/logo-from-file.png" alt="Figure 4-7: The MonoGame logo drawn to the game window."><figcaption><p><strong>Figure 4-7: The MonoGame logo drawn to the game window.</strong></p></figcaption></figure>
+
+When drawing with MonoGame, the graphics device needs to be prepared and told that it is going to receive data to pass to the GPU to render. When using the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) for rendering, we do this by using the [SpriteBatch.Begin](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch#Microsoft_Xna_Framework_Graphics_SpriteBatch_Begin_Microsoft_Xna_Framework_Graphics_SpriteSortMode_Microsoft_Xna_Framework_Graphics_BlendState_Microsoft_Xna_Framework_Graphics_SamplerState_Microsoft_Xna_Framework_Graphics_DepthStencilState_Microsoft_Xna_Framework_Graphics_RasterizerState_Microsoft_Xna_Framework_Graphics_Effect_System_Nullable_Microsoft_Xna_Framework_Matrix__)  method. We can't use the [SpriteBatch.Draw](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.Draw(Microsoft.Xna.Framework.Graphics.Texture2D,Microsoft.Xna.Framework.Vector2,Microsoft.Xna.Framework.Color)) method without first using the [SpriteBatch.Begin](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch#Microsoft_Xna_Framework_Graphics_SpriteBatch_Begin_Microsoft_Xna_Framework_Graphics_SpriteSortMode_Microsoft_Xna_Framework_Graphics_BlendState_Microsoft_Xna_Framework_Graphics_SamplerState_Microsoft_Xna_Framework_Graphics_DepthStencilState_Microsoft_Xna_Framework_Graphics_RasterizerState_Microsoft_Xna_Framework_Graphics_Effect_System_Nullable_Microsoft_Xna_Framework_Matrix__)  so the graphics device is set to a state ready to draw.  [SpriteBatch.End](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.End) must always be called when finished rendering with the [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch).  When the [End](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.End)` method is called, it is at this point that the batched draw calls are submitted to the GPU for rendering.

```

Comment:
Consider rewording as this was confusing to read.  I suspect what you are staying are two separate statements:

* When drawing you need to initialise the graphics device
* When using SpriteBatch, this does a lot of the required initialisation for you, so you do not need to.

But this also REQUIRES a note, that later when you draw 2D AND 3D, you HAVE to define graphics states.  This trips a lot of people up later as to why spritebatch messes up their rendering.

Reply:
Going to need to restructure this section. I think having this information first before writing the code example would be better than providing it after the code example.  Need to think about how to swap the two and make these adjustments

Reply:
Section has been restructured to better explain these concepts